V2.3/JSON - Node Red - Flows Required.json

```diff
@@ -1,16 +1,45 @@
 [
     {
-        "id": "ed455582310cee2d",
+        "id": "ec086ef3e6c65108",
         "type": "tab",
         "label": "SunSynk Gather",
         "disabled": false,
         "info": "",
         "env": []
     },
     {
-        "id": "ef22d493df1c5071",
+        "id": "bc1c0f273da7bbae",
+        "type": "hass-config",
+        "host": "http://192.168.0.12",
+        "port": "8123"
+    },
+    {
+        "id": "53bc65975025ce60",
+        "type": "server",
+        "name": "Home Assistant",
+        "version": 5,
+        "addon": false,
```

Comment:
Changing this to false will work with seperate docker install of HA and NR, just remember to set the private key from HA user, and use external IP with port map for hass-config if accessing the container directly does not work

Reply:
This still need to look at. I prefer the orrigional way for a few reasons, but am considering this